### PR TITLE
Add policy parameter for R4.0

### PR DESCRIPTION
**Pull request by tasket**
Issue https://github.com/QubesOS/qubes-issues/issues/3270

### CODE REVIEW

File: security/vm-sudo.md [hunk 1]
```diff
@@ -105,7 +105,8 @@ this for extra security.**
 1. Adding Dom0 "VMAuth" service:
 
         [root@dom0 /]# echo -n "/usr/bin/echo 1" >/etc/qubes-rpc/qubes.VMAuth
-        [root@dom0 /]# echo -n "\$anyvm dom0 ask" >/etc/qubes-rpc/policy/qubes.VMAuth
+        [root@dom0 /]# echo -n "\$anyvm dom0 ask,default_target=dom0" \
+        >/etc/qubes-rpc/policy/qubes.VMAuth
```
Comment (marmarek) [inline security/vm-sudo.md]: I know it was here already, but why `echo -n`, in both cases? Looks unnecessary.

Comment (tasket) [reply security/vm-sudo.md]: Don't know why. I'll remove it.

**Comment by andrewdavidwong** [comment]
Will this also work on 3.2? If not, you should create a new section, rather than *replacing* accurate 3.2 instructions with 4.0 instructions.

**Comment by marmarek** [comment]
Should be fine for R3.2 too, see https://github.com/QubesOS/qubes-issues/issues/3270#issuecomment-341479193

**Comment by andrewdavidwong** [comment]
Oh, ok! My mistake for missing that.

**Pull request merged**